.rubocop.yml

```diff
@@ -0,0 +1,29 @@
+Lint/UnusedMethodArgument:
+    AllowUnusedKeywordArguments: true
```

Comment:
The way I see it, there merely happen to be no other unused kwargs. But if there were, we'd just disable the cop there too.